### PR TITLE
chore(docs): update compatibility matrix

### PR DESCRIPTION
**Pull request by marciocadev**
Function(timeout) already implemented to tf-aws and awscdk
Function(memory) already implemented to tf-aws
Queue(timeout) already implemented to tf-aws and awscdk
Queue(pop) already implemented to sim, tf-aws and awscdk

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/04-reference/compatibility/compatibility.json [hunk 1]
```diff
@@ -1,669 +1,1751 @@
 {
```
Comment (Chriscbr) [inline docs/04-reference/compatibility/compatibility.json:1]: Can we undo the formatting changes? (it makes it harder to review the changes) We should handle this in a separate PR and add a dedicated lint command / build step

Comment (marciocadev) [reply docs/04-reference/compatibility/compatibility.json]: donoe, and adding queue.pop too

File: docs/04-reference/compatibility/compatibility.json [hunk 2]
```diff
@@ -192,11 +192,11 @@
...
     },
     "pop": {
-      "sim": {"implemented": false, "issue": 1385},
-      "tf-aws": {"implemented": false, "issue": 1385},
+      "sim": {"implemented": true, "issue": 1385},
+      "tf-aws": {"implemented": true, "issue": 1385},
       "tf-azure": {"implemented": false},
       "tf-gcp": {"implemented": false},
-      "aws-cdk": {"implemented": false, "issue": 1385}
+      "aws-cdk": {"implemented": true, "issue": 1385}
```
Comment (Chriscbr) [inline docs/04-reference/compatibility/compatibility.json:199]: I think we can remove the issue number for these

**Review by Chriscbr — COMMENTED** [review]
Looks good - I'll let @tsuf239 double check and give the final approval 🙂

**Review by tsuf239 — APPROVED** [review]
Thanks!

**Pull request merged**